### DISCUSSION TITLE
表情包样式调节

### DISCUSSION THREAD

**Discussion by stone0118** [walinejs/waline#1062] [Q&A] [answered]
问问我看你们这个吧表情包点开就很大，我的却很小，这个哪里调节呀，谢谢。电脑端看就更小了
![IMG_20220525_214308](https://user-images.githubusercontent.com/70432247/170277754-e72c705a-89a4-4891-af1e-69cdfb41f822.jpg)
![IMG_20220525_214905](https://user-images.githubusercontent.com/70432247/170278064-669ef87e-10d3-422d-85e7-473bc5c4c7e9.jpg)

**Comment by Mister-Hope** [comment]
我也不是很清楚，但你是不是有调整过系统字体？反正我这也不知道咋复现我们就先随便猜猜好了。

**Comment by lizheming** [comment, marked as answer]
html 上有一个 `font-size: 58%` 的样式，由于本身博客的话在 `body` 上有额外的 `font-size` 配置，所以这个设置对博客本身并不生效。但是 Waline 因为是基于 `--waline-font-size` 进行的布局，默认的 `--waline-font-size` 是 1rem，而 rem 是相较于 `html` 根节点进行设置的，所以这个 `58%` 就会对 Waline 的样式有影响。把这个 58% 的设置移除掉就会恢复正常大小了。
